src/simulation/power/simpleBattery/simpleBattery.cpp

```diff
@@ -59,6 +59,16 @@ void SimpleBattery::evaluateBatteryModel(PowerStorageStatusMsgPayload *msg) {
         this->storedCharge = 0;
     }
 
+    if(this->fault == true) {   
+        if(this->faultCapacityRatio < 0)
+        {
+            this->faultCapacityRatio = 1.0;
+        }
+        if(this->storedCharge > this->storageCapacity * this->faultCapacityRatio) {
+            this->storedCharge = this->storageCapacity * this->faultCapacityRatio;
+        }
```

Comment:
Could this be simplified and made more strict regarding acceptable values of `faultCapacityRatio`?
```
if (this->faultCapacityRatio < 0 || this->faultCapacityRatio > 1)
    bskLogger.bskLog(BSK_ERROR, "faultCapacityRatio should be between 0 and 1!");
else
    this->storedCharge = this->storageCapacity * this->faultCapacityRatio;
```

Reply:
I agree.  Right now this code silently changes the value if <0 and the user is not notified.  I like the more compact version that doesn't silently change values.

Reply:
@juan-g-bonilla , your code actually always set the stored charge to the max value.  I think the code should be:

```
if (this->faultCapacityRatio < 0 || this->faultCapacityRatio > 1) {
    bskLogger.bskLog(BSK_ERROR, "faultCapacityRatio should be between 0 and 1!");
} else if(this->storedCharge > this->storageCapacity * this->faultCapacityRatio) {
            this->storedCharge = this->storageCapacity * this->faultCapacityRatio;
        }
}
```